droidlet/memory/memory_filters.py

```diff
@@ -590,6 +590,39 @@ def _selfstr(self):
         return "Basic: (" + str(self.search_data) + ")"
 
 
+class BackoffFilter(MemoryFilter):
+    """
+    runs a sequence of Filters, passed into the __init__.  if nothing is returned from the 
```

Comment:
Can we rephrase this to something like - goes over the list of filters in __init__ and returns the first not None (memids, vals) tuple ?